cmd/resim/commands/client.go

```diff
@@ -49,8 +70,14 @@ func init() {
 	rootCmd.PersistentFlags().String(clientSecretKey, "", "Authentication credentials client secret")
 	rootCmd.PersistentFlags().String(devInteractiveClientKey, "", "Client ID for dev interactive login")
 	viper.SetDefault(devInteractiveClientKey, "Rg1F0ZOCBmVYje4UVrS3BKIh4T2nCW9y")
+	rootCmd.PersistentFlags().String(devNonInteractiveClientKey, "", "Client ID for dev non-interactive login")
+	viper.SetDefault(devNonInteractiveClientKey, "LLNl3xsbNLSd16gQyYsiEn3tbLDZo1gj")
 	rootCmd.PersistentFlags().String(prodInteractiveClientKey, "", "Client ID for prod interactive login")
 	viper.SetDefault(prodInteractiveClientKey, "gTp1Y0kOyQ7QzIo2lZm0auGM6FJZZVvy")
+	rootCmd.PersistentFlags().String(prodNonInteractiveClientKey, "", "Client ID for prod non-interactive login")
+	viper.SetDefault(prodNonInteractiveClientKey, "0Ip56H1LLAo6Dc6IfePaNzgpUxbJGyVI")
+	rootCmd.PersistentFlags().String(usernameKey, "", "username for non-interactive login")
```

Comment:
```suggestion
	rootCmd.PersistentFlags().String(usernameKey, "", "username for non-interactive login")
```
Thought: you could do a set mutually exclusive for some of these to give validation

Reply:
You're right, I should check only one set is set